docs/research/Initial-Research.md

```diff
@@ -0,0 +1,273 @@
+# Initial Research for Internal Systems 
+## Website Technologies
+#
+<br />
+
```

Comment:
Maybe pop an introduction in as to what decision this research is being used for within the Internal Systems project?